components/camel-jetty-common/src/main/java/org/apache/camel/component/jetty/JettyHttpComponent.java

```diff
@@ -236,10 +237,11 @@ protected Endpoint createEndpoint(String uri, String remaining, Map<String, Obje
             endpoint.setBinding(binding);
         }
         // prefer to use endpoint configured over component configured
-        if (jettyBinding == null) {
+        // unused
+        /*if (jettyBinding == null) {
```

Comment:
I think we can remove this block of code if it's unused. It would make it easier for us when/if bisecting the code in case of problems.